app/views/people/_actions_show.html.haml

```diff
@@ -29,8 +29,6 @@
 - if entry == current_user
   - if entry.encrypted_password?
```

Comment:
Jetzt wo du das in einer Ability hast, könntest du diesen Check durch eine CanCan Assertion ersetzen.